### PR TITLE
Improving debugging experience by .bat files

### PR DESCRIPTION
**Pull request by ivanboika**
My debugging experience was full of restarting our server or even both parts (client and server). Sometimes it was mind-numbing and I decided to automize it. Thanks Konstantin for the idea of .bat files

### CODE REVIEW

**Comment by memorisecodead** [comment]
It's a good thing I did. This has been discussed for a long time and the .bat file will finally appear. Now I will be able to not use my .bat file :)

**Review by memorisecodead — APPROVED** [review]

**Comment by ivanboika** [comment]
> It's a good thing I did. This has been discussed for a long time and the .bat file will finally appear. Now I will be able to not use my .bat file :)

File: Docs/Improved debugging.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+If you do not want to use your terminal enormous amount of times while you are debugging or testing new features it can help you
+
+You can use .bat files to run server or server & client on your own PC already with arguments and by 2 clicks of mouse
+
+#### 1. Creation of server.start .bat file
+1. Create an empty .txt file where you wish
+2. Copy the next piece of text and paste it into already created .txt file
+
+	```c++
```
Comment (Orange-hanter) [inline Docs/Improved debugging.md:11]: is that c++ code?

**Review by Orange-hanter — APPROVED** [review]

File: Docs/Improved debugging.md (hunk 1 above)
Comment (memorisecodead) [reply Docs/Improved debugging.md:11]: I think, yes

**Review by slavexx — APPROVED** [review]

File: Docs/Improved debugging.md (hunk 1 above)
Comment (ivanboika) [reply Docs/Improved debugging.md:11]: Of course no, I just found this tip in our docs, I decided to underline text and it was only way that knew
Maybe I'll find better ways about this point

**Pull request merged**